api/src/owner_rpc.rs

```diff
@@ -256,7 +256,7 @@ pub trait OwnerRpc {
 			  "confirmation_ts": "2019-01-15T16:01:26Z",
 			  "confirmed": true,
 			  "creation_ts": "2019-01-15T16:01:26Z",
-			  "fee": null,
+			  "fee_fields": null,
```

Comment:
Just so we know this is a breaking change with the current API.

Reply:
This also breaks compatibility with older wallets that already have tx log entries stored in their wallet db. This is not intended, will fix.